atc/atccmd/command.go

```diff
@@ -1015,8 +1016,9 @@ func (cmd *RunCommand) backendComponents(
 	}
 
 	rateLimiter := db.NewResourceCheckRateLimiter(
-		dbConn,
+		rate.Limit(cmd.MaxChecksPerSecond),
```

Comment:
The existing behaviour is:

* 0 or unset ==> default limit
* -1 ==> no limit
* anything else ==> static limit

I think we're just missing the `-1` case

Reply:
ah, yep - fixed!